docs/GettingStarted.md

```diff
@@ -256,7 +256,7 @@ The `scan-result.yml` file can now be used as input for the reporter to generate
 generate both a static HTML report and an Excel report use:
 
 ```bash
-cli/build/install/ort/bin/ort report -f StaticHtml,Excel -i [scanner-output-path]/mime-types/scan-result.yml -o [reporter-output-path]/mime-types
+cli/build/install/ort/bin/ort report -f StaticHtml,Excel -i [scanner-output-path]/scan-result.yml -o [reporter-output-path]/mime-types
```

Comment:
the mime-types dir is something we could get rid of as well (+2 more occurences right below)
[reporter-output-path]/mime-types

Reply:
True, done.